testfixtures/logcapture.py

```diff
@@ -106,17 +107,17 @@ def uninstall_all(cls):
         "This will uninstall all existing :class:`LogHandler` objects."
         for i in tuple(cls.instances):
             i.uninstall()
-        
+
```

Comment:
It's PEP8. Trailing spaces / empty lines do not contribute to the code, so they're useless bytes.